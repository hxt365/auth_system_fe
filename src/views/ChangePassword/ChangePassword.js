import React from 'react';

function ChangePassword() {
  return <p>ChangePassword</p>;
}

export default ChangePassword;
