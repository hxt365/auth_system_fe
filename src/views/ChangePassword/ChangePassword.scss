.change-password-form {
  max-width: 60rem;
  margin-left: calc((100% - 60rem) / 2) !important;
}
