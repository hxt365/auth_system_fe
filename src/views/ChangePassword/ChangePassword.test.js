import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import ChangePassword from './ChangePassword';

describe('test ChangePassword', () => {
  it('test snapshot', () => {
    const wrapper = shallow(<ChangePassword />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
