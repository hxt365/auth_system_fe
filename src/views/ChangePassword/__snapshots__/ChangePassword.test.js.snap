// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test ChangePassword test snapshot 1`] = `
<section
  className="change-password"
>
  <h2>
    Important: Never tell your girlfriend the password!
  </h2>
  <Form(ChangePasswordForm) />
</section>
`;
